guides/user/pipeline/pipeline-templates/instantiate.md

```diff
@@ -91,7 +91,7 @@ for which you're providing values, and write that value.
    ```
 You can code each value by hand in the pipeline JSON that you create. You can
 also generate the JSON and populate the values programatically. For simplicity
-This doc describes doing it by hand.
+This doc describes doing it by hand. <!-- This sentence sounds like "This doc" should be hyperlinked but I don't see anything. I wanted to bring attention to this in case there is supposed to be a link -->
```

Comment:
It's referring to the doc that it's in--it's self referential! But I agree it's a little ambiguous. I thing if it were at the beginning of the article, it would be clear. I'll suggest an edit.

Reply:
Actually, @stephly07 , in the interest of getting as many of these PRs merged as we can, and because it's not too terribly ambiguous, I'm going to let it ride for now.

Reply:
Sounds good!